src/ServicePulse.Host/vue/src/components/PageHeader.vue

```diff
@@ -36,47 +29,47 @@ const displayDanger = computed(() => {
 
       <div id="navbar" class="navbar navbar-expand-lg">
         <ul class="nav navbar-nav navbar-inverse">
-          <li :class="{ active: subIsActive('/dashboard', true) }">
+          <li :class="{ active: isRouteSelected(routeLinks.dashboard) }">
             <RouterLink :to="routeLinks.dashboard">
               <i class="fa fa-dashboard icon-white" title="Dashboard"></i>
               <span class="navbar-label">Dashboard</span>
             </RouterLink>
           </li>
-          <li :class="{ active: subIsActive('/a/#/endpoints') }">
+          <li>
             <a :href="routeLinks.heartbeats">
               <i class="fa fa-heartbeat icon-white" title="Heartbeats"></i>
               <span class="navbar-label">Heartbeats</span>
               <span v-if="stats.number_of_failed_heartbeats > 0" class="badge badge-important">{{ stats.number_of_failed_heartbeats }}</span>
             </a>
           </li>
-          <li v-if="useIsMonitoringEnabled()" :class="{ active: subIsActive('/a/#/monitoring') || subIsActive('/a/#/monitoring/endpoint') }">
+          <li v-if="useIsMonitoringEnabled()">
             <a :href="routeLinks.monitoring">
               <i class="fa pa-monitoring icon-white" title="Monitoring"></i>
               <span class="navbar-label">Monitoring</span>
               <span v-if="stats.number_of_disconnected_endpoints > 0" class="badge badge-important">{{ stats.number_of_disconnected_endpoints }}</span>
             </a>
           </li>
-          <li :class="{ active: subIsActive('/failed-messages') }">
+          <li :class="{ active: isRouteSelected(routeLinks.failedMessage.root) }">
             <RouterLink :to="routeLinks.failedMessage.root">
               <i class="fa fa-envelope icon-white" title="Failed Messages"></i>
               <span class="navbar-label">Failed Messages</span>
               <span v-if="stats.number_of_failed_messages > 0" class="badge badge-important">{{ stats.number_of_failed_messages }}</span>
             </RouterLink>
           </li>
-          <li :class="{ active: subIsActive('/a/#/custom-checks') }">
```

Comment:
I completely removed this code, given this would never be active in the vue side

Reply:
And again